README.md

```diff
@@ -43,6 +43,8 @@ DOCKER_BUILDKIT=1 docker build --tag=janus_aggregation_job_driver --build-arg BI
 DOCKER_BUILDKIT=1 docker build --tag=janus_collection_job_driver --build-arg BINARY=collection_job_driver .
 ```
 
+Alternately, run `docker buildx bake` to build images in parallel.
```

Comment:
Is there any reason not to suggest only the `docker buildx bake` approach? i.e. is there a use-case for building individual images anymore? If not, I think we can just drop the old instructions entirely.